### DISCUSSION TITLE
Z世代有個秘密：我也沒那麼懂科技！遇到設備技術問題，羞恥感比40歲高10倍

### DISCUSSION THREAD

**Discussion by peggy0329** [1101-GS4719-C/Python-Community-GS4719-C#958] [General]
文章連結：https://www.bnext.com.tw/article/73305/gen-z-tech-shame-202212

我真的覺得這個文章打中我們這個世代人的痛處！！
也是我們現在年輕人該注重的議題
科技讓我們好像不會一點技術就感到有點羞愧
但生在一個科技便利的時代，真的不代表我們對任何技術的了解
還記得之前實習時，主管請我將影印機資料傳真到我電腦
我的天，我研究了一小段時間還是不太確怎麼用，壓力真的好大zzz 
大家有沒有在職場上也遇過類似的事情呢？

**Comment by moonnight3503** [comment]
確實，有時候家人問我一些我沒接觸過的科技，如果我不知道他們就會很驚訝我怎麼會不知道==

**Comment by Zoe06060606** [comment]
的確 有時候長輩詢問科技的問題的時候都會帶著年輕人就是比較懂科技的語氣把問題順理成章的丟過來 真的會有壓力

**Comment by Floriaand** [comment]
常常有長輩問我一些電腦的問題，我跟他們說我也不會的時候就覺得很尷尬

**Comment by Evan0979** [comment]
最近地一次買apple的產品,沒用過ios系統,每次問同學的時候都會被笑嗚嗚嗚

**Comment by swerfg03joe** [comment]
Z世代並沒有比較懂科技，他們是對科技的接受度很高，且很願意去學習自己不懂的事物，不管是新軟體或者是新的硬體，而早期的人並不像Z世代處在資訊爆炸的世界，多數科技都是近20~30年出現並蓬勃發展的，Z世代本身就要能適應這瞬息萬變的世界，才能順利度過求學的每個階段，自然會讓人覺得比較懂科技

**Comment by BlahFish1006** [comment]
我覺得這就和語言一樣，生長在一個用中文溝通的環境，那我們就自然需要會對中文熟悉才能順利溝通，
而Ｚ世代就是充斥著快速演變的科技的環境，所以自然會需要對科技的上手相對熟悉才能順利生存，
個人覺得不會用也不需要羞恥，就只是因為沒用過、沒接觸過，所以對那個系統不熟悉罷了。

**Comment by WEI-CHEN-CHUN** [comment]
科技日益月新，不同的科技都得花時間學習，有些人本來就學的較慢
